src/spaceone/dashboard/service/domain_dashboard_service.py

```diff
@@ -49,7 +49,8 @@ def create(self, params):
         domain_dashboard_vo = self.domain_dashboard_mgr.create_domain_dashboard(params)
 
         version_keys = ['layouts', 'dashboard_options', 'dashboard_options_schema']
-        if set(version_keys) <= params.keys():
+        if any(key for key in params
```

Comment:
I suggest improving the code here as follows.
`any(set(version_keys) & set(params.keys()))`